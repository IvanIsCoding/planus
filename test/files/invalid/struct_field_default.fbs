struct Foo {
  x: uint8 = 1;
}