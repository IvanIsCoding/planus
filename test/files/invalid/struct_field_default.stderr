[0m[1m[38;5;9merror[0m[1m: [0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/struct_field_default.fbs:2:14
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   x: uint8 = [0m[31m1[0m;
  [0m[34mâ”‚[0m              [0m[31m^[0m [0m[31mStruct fields cannot have default values[0m

Error: Bailing because of errors
